ansible_builder/main.py

```diff
@@ -39,7 +39,19 @@ def __init__(self,
                  container_runtime=constants.default_container_runtime,
                  output_filename=None,
                  no_cache=False,
-                 verbosity=constants.default_verbosity):
+                 verbosity=constants.default_verbosity,
+                 galaxy_keyring=None,
+                 galaxy_required_valid_signature_count=None,
+                 galaxy_ignore_signature_status_code=None):
+        """
+        :param str galaxy_keyring: GPG keyring file used by ansible-galaxy to opportunistically validate collection signatures.
+        :param str galaxy_required_valid_signature_count: Number of sigs (prepend + to disallow no sig( required for ansible-galaxy to accept collections.
```

Comment:
Still have a ( here that should be a ).

Reply:
oh .... did not notice I copy pasted that in 5 locations, thanks!